voice-client-javascript/assets/admin/actions.private.js

```diff
@@ -0,0 +1,83 @@
+class Actions {
+  constructor(client, options) {
+    this.client = client;
+    this.options = options;
+  }
+
+  async initialize() {
+    let env = {};
+    console.log("Creating TwiML Application");
+    let results = await this.createTwimlApp(this.options);
+    env = Object.assign(env, results);
+    const voiceUrl = `https://${this.options.DOMAIN_NAME}/client-voice-twiml-app`;
+    console.dir(env);
+    console.log(
+      `Wiring up TwiML Application ${env.TWIML_APPLICATION_SID} to the function: ${voiceUrl}`
+    );
+    await this.updateTwimlAppVoiceUrl({
+      twimlApplicationSid: env.TWIML_APPLICATION_SID,
+      voiceUrl
+    });
+    console.log("Generating new REST API Key");
+    results = await this.generateNewKey(this.options);
+    env = Object.assign(env, results);
+    const number = await this.chooseLogicalCallerId();
+    results = await this.setCallerId({ number });
+    env = Object.assign(env, results);
+    env.INITIALIZED = "voice-client-quickstart";
```

Comment:
Can we prefix this in some way to make sure it doesn't break other applications.

Reply:
This applies throughout. Normally I'm not concerned about this but since you are modifying env variables automatically, I'm a bit more concerned about it because it doesn't create visibility for people

Reply:
Added tests for the controllers and unit tests for the actions. Working through the status unit tests now.